.yamlfmt.yml

```diff
@@ -2,6 +2,8 @@
 
 # https://github.com/google/yamlfmt
 
+gitignore_excludes: true
```

Comment:
ignore everything that is in .gitignore. Otherwise, it would check all contents of the virtual environment.